client/actions/main.ts

```diff
@@ -236,7 +236,8 @@ const save = (original, updates, withConfirmation = true) => (
 
         switch (itemType) {
         case ITEM_TYPE.EVENT:
-            promise = dispatch(eventsUi.save(original, updates, confirmation));
+            confirmation = false;
```

Comment:
Confirmation **may** be required when saving an Event (that's where `withConfirmation` comes into play).